OpenOversight/app/auth/views.py

```diff
@@ -319,7 +319,17 @@ def edit_user(user_id):
                 form.populate_obj(user)
                 db.session.add(user)
                 db.session.commit()
+
+                # automatically send a confirmation email when approving an unconfirmed user
+                if (
+                    not (current_user.approved)
```

Comment:
What's the purpose of this conditional? It seems that this would only run if the user who is performing the editing has not been approved.

Reply:
Oops, that's mb! I meant to check that the user was previously unapproved and is now approved. Let me fix that sorry